misk-hibernate/src/main/kotlin/misk/hibernate/ReflectionQueryFactory.kt

```diff
@@ -94,8 +94,14 @@ internal class ReflectionQuery<T : DbEntity<T>>(
   fun addConstraint(path: List<String>, operator: Operator, value: Any?) {
     when (operator) {
       Operator.EQ -> {
```

Comment:
It might be better to use a new operator for this, e.g. `EQ_OR_IS_NULL`. I don't think this should break anything in `EQ`, but just want to be sure.